index.js

```diff
@@ -91,6 +91,57 @@ exports.register = function (server, options, next) {
     }
   });
 
+  server.route({
+    method: 'GET',
+    path: '/events/{eventId}',
+    handler: function (request, reply) {
+      reply(openGraphTemplate({
+        openGraphProperties: request.app.context.preload
+      }));
+    },
+    config: {
+      plugins: {
+        senecaPreloader: {
+          handler: 'seneca-event-preloader'
+        }
+      },
+    },
+  });
+  
+  server.route({
+    method: 'GET',
+    path: '/events/{eventId}/sessions',
+    handler: function (request, reply) {
+      reply(openGraphTemplate({
+        openGraphProperties: request.app.context.preload
+      }));
+    },
+    config: {
+      plugins: {
+        senecaPreloader: {
+          handler: 'seneca-event-preloader'
+        }
+      },
+    },
+  });
+  
+  server.route({
+    method: 'GET',
+    path: '/events/{eventId}/confirmation',
+    handler: function (request, reply) {
+      reply(openGraphTemplate({
```

Comment:
Do we really need to bother with open graph on the confirmation page?

Reply:
The url is unique, so I'd prefer too

Reply:
We should probably redirect to the sessions page if someone gets to this page and doesn't have an order. We already redirect to LoginOrRegister if they're not logged in.

Reply:
Not sure what's a good solution here. A navguard would be nice, but that means injecting the store into it. 
Can a navguard be bypassed if you come from another url, (like, normal flow) and only triggered on "refresh" ?

Reply:
It should be possible. The nav guard gets `from` and `to` passed into it. On refresh, we should be able to check that there is no `from`. Need to check exactly what it is in that scenario, but I don't see how it couldn't work.